drift_dev/test/analysis/resolver/dart/table_test.dart

```diff
@@ -142,6 +158,42 @@ class InvalidConstraints extends Table {
     });
   });
 
+  group("class name inflection", () {
+    test("singularizes table name that end in -ies", () async {
+      final result = await findTable('Properties');
+      final table = result!.result as DriftTable;
+
+      expect(table.nameOfRowClass, 'Property');
+    });
+
+    test("singularizes table name that end in -s", () async {
+      final result = await findTable('Cats');
+      final table = result!.result as DriftTable;
+
+      expect(table.nameOfRowClass, 'Cat');
+    });
+
+    test("singularizes table name that end in -es", () async {
+      final result = await findTable('Messages');
+      final table = result!.result as DriftTable;
+
+      expect(table.nameOfRowClass, 'Message');
+    });
+
+    test("singularizes table name that end in -os", () async {
+      final result = await findTable('Pianos');
+      final table = result!.result as DriftTable;
+
+      expect(table.nameOfRowClass, 'Piano');
+    });
+
+    test("singularizes table name that end in -s", () async {
+      final result = await findTable('Socks');
+      final table = result!.result as DriftTable;
+
+      expect(table.nameOfRowClass, 'Sock');
```

Comment:
Could you move this into the first `singularizes table name that end in -s` test so that the test verifies both cases (or just remove this since it seems to test the same thing if I'm not overlooking anything)? Having two tests with the same name might be confusing.

Reply:
I must have gotten carried away with the test cases. I removed the duplicated test case.